addons/udes_security/models/allowed_file_type.py

```diff
@@ -21,6 +42,9 @@ class AllowedFileType(models.Model):
     )
     description = fields.Char("Description")
     active = fields.Boolean("Active?", default=True)
+    mimetype_ids = fields.Many2many(
```

Comment:
Many2many fields generally are defined where will be a possibility that for 2 different filetypes will have common mimetypes. Is it a scenario where same mimetype can be in 2 different filetypes, if not i would suggest one2many field not many2many.

Reply:
In our allowed types we have `tif` and `tiff`, which both have a mime type of `image/tiff` so I think many to many is the best way to handle it. There could be others I'd imagine: for example some systems might use `text/plain` for js and css files.

Reply:
Yeah makes sense thats why asked do we have common mimetypes for different file types?